test_data/yast/encryption/full_alp.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
time to create `test_data/yam/` and perhaps this time avoid to create subfolders using proper naming, `full_disk_encryption_alp.yaml` for example ?

Reply:
Deleted this in the `yast` folder and merged the content into the `YAML_TEST_DATA`. Seems to make more sense.